app/src/main/res/layout/fragment_register.xml

```diff
@@ -0,0 +1,159 @@
+        android:padding="@dimen/email_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/email_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+
+
+        />
+
+    <EditText
+        android:id="@+id/password"
+        android:layout_width="@dimen/pass_width"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:drawableLeft="@drawable/ic_pass"
+        android:hint="@string/password"
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="numberPassword"
+        android:padding="@dimen/pass_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/pass_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+
+        />
+
+    <EditText
+        android:id="@+id/confirm_password"
+        android:layout_width="@dimen/confirm_pass_width"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:drawableLeft="@drawable/ic_pass"
+        android:hint="@string/confirm_password"
+        android:textColorHint="@color/color_FDFBFB"
+        android:inputType="numberPassword"
+        android:padding="@dimen/confirm_pass_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/confirm_pass_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+        />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/height_19"
+        android:layout_marginTop="@dimen/marginTop_25"
+        android:orientation="horizontal"
+        android:layout_marginLeft="@dimen/marginleft_75"
+        >
+
+        <CheckBox
+            android:id="@+id/cb_reg"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:buttonTint="@color/color_fff"
+
+
+            />
+
+        <TextView
+            android:id="@+id/txt"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_toRightOf="@id/cb_reg"
+            android:text="@string/i_agree_to_the"
+            android:textColor="@color/color_FFF7F2F2"
+
+
+            />
+        <TextView
+            android:id="@+id/terms_conditions"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/terms_and_conditions"
+            android:textColor="@color/color_FFF7F2F2"
+            android:layout_toRightOf="@id/txt"
+
+            />
+
+
+    </LinearLayout>
+    <View
+        android:layout_width="@dimen/view_width"
+        android:layout_height="@dimen/view_height"
+        android:background="@color/color_e6e4e4"
+        android:layout_marginLeft="@dimen/marginleft_173"
+        android:layout_marginBottom="@dimen/marginbott_123"
+
+        />
+
+    <Button
```

Comment:
button is not centered horizontally and very far from the checkbox "i agree" 
please review against the design again